### PR TITLE
Character/scientist

### LINKED ISSUE #36: Add model to Scientists
Create a prefab of the scientist with animations.

### LINKED ISSUE #39: Scientists AI
Create a scientist AI that runs away from the player

### PR DESCRIPTION
**Pull request by claudiarmamede**

### CODE REVIEW

File: Assets/Scripts/Player/Player.cs [hunk 1]
```diff
@@ -53,4 +57,13 @@ void Awake()
             this.Die();
         }
     }
+
+    public void becomeEvil(){
+        pointsToEvil--;
+
+        if(pointsToEvil == 0) {
+            //muda de cor
+        }
```
Comment (CajoAlbuquerque) [inline Assets/Scripts/Player/Player.cs:66]: The texture is already in the repo. Changing the albedo of the Material programatically should be enough. We can also create a whole new material with the evil tezture and just swap materials mid game. No need to worry about smooth transitions! I guess the latter options is easier. Can you implement this in this branch right away?

**Review by CajoAlbuquerque — APPROVED** [review]
I am approving because everything seems fine. I have not tested the game as that would drain my battery really quick. Consider the above comment, but if you think it is better to implement it in another branch than I'm ok with it as well

**Review by bernas670 — CHANGES_REQUESTED** [review]
I noticed a little detail, when the scientist is dying he keeps moving away from the player. I had a similar problem when adding animations to the soldier. When you use the `communicateDeath` just set a `boolean` in the `ScientistController` so he does not move.
Other than that great job!

**Comment by BernardoCMoreira** [comment]
> I noticed a little detail, when the scientist is dying he keeps moving away from the player. I had a similar problem when adding animations to the soldier. When you use the `communicateDeath` just set a `boolean` in the `ScientistController` so he does not move.
> Other than that great job!

Yeah thanks bernas! It's done

**Review by bernas670 — APPROVED** [review]
Lgtm! :ship:

**Pull request merged**